src/Microsoft.Health.Fhir.Api/Features/Filters/ValidateExportRequestFilterAttribute.cs

```diff
@@ -111,7 +120,10 @@ private bool IsValidBasicExportRequestParam(string paramName)
 
         private bool IsValidAnonymizedExportRequestParam(HttpRequest request, string paramName)
         {
-            if (request.Path.StartsWithSegments(DefaultExportRequestPath, StringComparison.InvariantCultureIgnoreCase))
+            if (AnonymizedExportRequestPathRegexs.Any(pathRegex => Regex.IsMatch(
+                input: request.Path,
+                pattern: pathRegex,
+                options: RegexOptions.IgnoreCase | RegexOptions.CultureInvariant)))
```

Comment:
This check isn't needed anymore since anonymized export is now available for all the types of export. The route filter will have removed any that don't fit this pattern.

Reply:
Got it, removed this check.